examples/subplots_axes_and_figures/custom_figure_class.py

```diff
@@ -1,27 +1,55 @@
 """
 ===================
-Custom Figure Class
+Custom Figure class
 ===================
 
-You can pass a custom Figure constructor to figure if you want to derive from
-the default Figure.  This simple example creates a figure with a figure title.
+You can pass a custom Figure class to `.pyplot.figure` if you want to change
+the default behavior of the `.Figure`.
+
+This example defines a `.Figure` subclass ``WatermarkFigure`` that accepts an
+additional parameter ``watermark`` to display a custom watermark text. The
+figure is created using the ``FigureClass`` parameter of `.pyplot.figure`.
+The additional ``watermark`` parameter is passed on to the subclass
+constructor.
 """
 
 import matplotlib.pyplot as plt
 from matplotlib.figure import Figure
+import numpy as np
+
 
+class WatermarkFigure(Figure):
+    """A figure with a text watermark."""
 
-class MyFigure(Figure):
-    def __init__(self, *args, figtitle='hi mom', **kwargs):
-        """
-        custom kwarg figtitle is a figure title
-        """
+    def __init__(self, *args, watermark=None, **kwargs):
         super().__init__(*args, **kwargs)
-        self.text(0.5, 0.95, figtitle, ha='center')
+
+        if watermark is not None:
+            bbox = dict(boxstyle='square', lw=3, ec='gray',
+                        fc=(0.9, 0.9, .9, .5), alpha=0.5)
```

Comment:
I don't really care whether you use 0.9 or .9, but at least be consistent :) (and same with 0.5 here and below)